packages/block-library/src/social-links/save.js

```diff
@@ -10,12 +10,18 @@ import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 export default function save( props ) {
 	const {
-		attributes: { iconBackgroundColorValue, iconColorValue, size },
+		attributes: {
+			iconBackgroundColorValue,
+			iconColorValue,
+			itemsJustification,
+			size,
+		},
 	} = props;
 
 	const className = classNames( size, {
 		'has-icon-color': iconColorValue,
 		'has-icon-background-color': iconBackgroundColorValue,
+		[ `items-justified-${ itemsJustification }` ]: itemsJustification,
```

Comment:
Did we discuss the naming of this CSS class? I seem to vaguely recall that we did, but can't recall where we ended. I would've probably preferred something like `has-justification-left` — but maybe there was already an existing CSS class out in the wild?